07-systems_access_policy.md

```diff
@@ -97,7 +97,7 @@ Access to Health Note systems and applications is limited for all users, includi
 ## 7.7 Automatic Logoff
 
 1. Users are required to make information systems inaccessible by any other individual when unattended by the users (ex. by using a password protected screen saver or logging off the system).
-2. Information systems automatically log users off or lock with password protected screen saver systems after 15 minutes of inactivity.
+2. Information systems automatically log users off or lock with password protected screen saver systems after 10 minutes of inactivity.
 3. The Security Officer pre-approves exceptions to automatic log off requirements.
 
 ## 7.8 Employee Workstation Use
```

Comment:
For #9 below, I don't see a way to turn that on with JAMF. Do you know how we can turn that on?

Reply:
For #10, we don't have that on for MacOS at least. Do we need to turn that on with JAMF?

Reply:
For 9, I will have to check if it is supported in JAMF now.

For 10, There isn't a way to turn this on, we can manually force updates across computers that are communicating. We can update it to remove MacOS updates on that line and add that MacOS users are reminded monthly to install the latest updates.

Reply:
Work tracked in https://healthnote.atlassian.net/browse/DEVOPS-2258. Added custom profile to Blueprint via iMazing profile editor to enable login message and MacOS automatic updates